src/Views/HardwareView.vala

```diff
@@ -280,6 +282,23 @@ public class About.HardwareView : Gtk.Grid {
 
         return 0;
     }
+
+    private string get_storage_type () {
+        string storage = "";
+        try {
+            UDisks.DriveSkeleton drive = new UDisks.DriveSkeleton ();
+            int rotational = drive.rotation_rate;
+            if (rotational == 0) {
+                storage = "(SSD)";
```

Comment:
These strings should probably be translatable. So it should be `storage = _("(SSD)");`. Don't forget the HDD as well :)